.eslintrc.json

```diff
@@ -84,7 +84,9 @@
                 "react/display-name": "off",
```

Comment:
I think in the current implementation we don't need `stories/index.ts`

Reply:
removed it now, that the folders import works